js/packages/cli/src/helpers/various.ts

```diff
@@ -178,9 +178,38 @@ export async function getCandyMachineV2Config(
   }
 
   if (endSettings) {
```

Comment:
see my note in the PR description. a cleaner solution would be to change `endSettingType` to expect a string and modify the object sent to the rust contract.